src/components/button/index.md

```diff
@@ -5,7 +5,19 @@ description: Base button component
 
 <script setup>
   import pButton from './Button.vue'
+  import pSpinner from '../spinner/spinner.vue'
```

Comment:
Build failed

It should be written in Capital case, `../spinner/Spinner.vue`.

Reply:
done